_posts/2022-08-19-kubeflow-user-survey-2022.md

```diff
@@ -17,7 +17,7 @@ The survey was comprised of 24 questions (multiple choice and freeform). It ran
 ## Key Takeaways
 
 - 85% of the users deploy more than one Kubeflow component
-- The top 3 Kubeflow components are Pipelines (89%), Notebooks (75%), and KServe (48%)
+- The top 3 Kubeflow components are Pipelines (89%), KServe(Formally KFServing) (82%), Notebooks (75%).
```

Comment:
Based on the conversation on [slack](https://kubeflow.slack.com/archives/C7REE0ETX/p1664931718646089?thread_ts=1664921666.407219&cid=C7REE0ETX), I like to suggest showing both combined and distinct result for KServe/KFServing. What do you think think about below?

cc @yuzisun @jbottum 

 ```suggestion
- The top 3 Kubeflow components are Pipelines (89%), KServe (Formally KFServing) (82% - a combination of KServe 48.3% and KFServing 33.3%), Notebooks (75%).
```